libindy/src/services/wallet/mod.rs

```diff
@@ -1028,6 +1035,38 @@ mod tests {
         wallet_service.close_wallet(handle).unwrap();
     }
 
+    #[test]
+    fn wallet_service_open_wallet_works_for_two_wallets_with_same_ids_but_different_paths() {
+        _cleanup();
+
+        let wallet_service = WalletService::new();
+
+        let config_1 = Config{
+            id: String::from("wallet_1"),
+            storage_type: None,
+            storage_config: None,
+        };
+
+        wallet_service.create_wallet(&config_1, &RAW_CREDENTIAL, (&RAW_KDD, &RAW_MASTER_KEY)).unwrap();
+        let handle_1 = wallet_service.open_wallet(&config_1,&RAW_CREDENTIAL).unwrap();
+
+        let config_2 = Config{
+            id: String::from("wallet_2"),
```

Comment:
I don't see how this tests two wallets with the same ID. The two wallets here have different IDs.

Reply:
fixed